functions/Set-DbaStartupParameter.ps1

```diff
@@ -80,6 +82,14 @@ Overrides the default behaviour and replaces any existing trace flags. If not tr
 Pass in a previously saved SQL Instance startUpconfig
 using this parameter will set TraceFlagsOverride to true, so existing Trace Flags will be overridden
 
+.PARAMETER Offline
+Setting this switch will try perform the requested actions without conntect to the SQL Server Instance, this will speed things up if you know the Instance is offline.
+
+Note, that with this switch Paths will be ignored unless you also speficy TrustPath
+
+.PARAMETER TrustPath
```

Comment:
I'd just use a `-Force` parameter as we do with other commands. Same rule applies for TrusthPath as it would Force.

Reply:
Switched Force for TrustPath in latest push.